@ecomplus/storefront-components/src/html/ProductCard.html

```diff
@@ -85,6 +85,15 @@
         </p>
       </slot>
 
+      <slot
+        name="upon-request"
```

Comment:
```suggestion
        name="price-on-request"
```

Acho que esse nome `upon-request` não diz muita coisa não...

Reply:
Ou talvez:

```suggestion
        name="without-price"
```

Se pá faz mais sentido, produto sem preço não precisa ser necessariamente "sob consulta"

Reply:
Tinha pensado nisso também, colocar dessa forma.